apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/AppController.java

```diff
@@ -141,6 +144,7 @@ public App create(@Valid @RequestBody AppModel appModel) {
 
   @PreAuthorize(value = "@permissionValidator.isAppAdmin(#appId)")
   @PutMapping("/{appId:.+}")
+  @ApolloAuditLog(type = OpType.UPDATE, name = "user-command.app.update")
```

Comment:
```suggestion
  @ApolloAuditLog(type = OpType.UPDATE, name = "app.update")
```


if in openapi, use `openapi.app.update`